apps/frontend/src/components/global/ExportCSVModal.vue

```diff
@@ -62,14 +62,13 @@ import LinkItem from '@/components/global/sidebaritems/IconLinkItem.vue';
 
 import {Filter, FilteredDataModule} from '@/store/data_filters';
 import ObjectsToCsv from 'objects-to-csv';
-import {HDFControlSegment} from 'inspecjs';
-import {ContextualizedControl} from 'inspecjs/dist/context';
 import {EvaluationFile, ProfileFile} from '../../store/report_intake';
 import {Prop} from 'vue-property-decorator';
 import {InspecDataModule} from '../../store/data_store';
-import {ControlDescription} from 'inspecjs/dist/generated_parsers/v_1_0/exec-json';
 import _ from 'lodash';
 import {saveSingleOrMultipleFiles} from '@/utilities/export_util'
+import {HDFControlSegment, ContextualizedControl} from 'inspecjs';
+import {ControlDescription} from 'inspecjs/src/versions/v_1_0/exec_json';
```

Comment:
All imports of `inspecjs` should happen directly from `inspecjs` and not from `src`. The `src` path will not exist on the production build of `inspecjs` and even though it would work in this case I'd like to get into the habit of us always using the external interface to inspecjs. In this can you can access it by importing `ExecJSON` and then the type is `ExecJSON.ControlDescription`.